app/assets/javascripts/application.js

```diff
@@ -19,4 +19,18 @@ $(document).ready(function(){
   $('.select2').select2({
     theme: 'bootstrap'
   });
-});
\ No newline at end of file
+  
+  
+  // Fix for lack of a11y in Select2
```

Comment:
Very nice for taking this on. ❤️ 

Properly accessible select boxes are a real challenge!!